mava/utils/jax_training_utils.py

```diff
@@ -58,13 +61,23 @@ def compute_running_mean_var_count(
     delta = batch_mean - mean
     tot_count = count + batch_count
 
-    new_mean = mean + delta * batch_count / tot_count
+    mean = mean + delta * batch_count / tot_count
     m_a = var * count
     m_b = batch_var * batch_count
     M2 = m_a + m_b + jnp.square(delta) * count * batch_count / tot_count
-    new_var = M2 / tot_count
+    var = M2 / tot_count
+    std = jnp.sqrt(var)
     new_count = tot_count
-    new_std = jnp.sqrt(new_var)
+
+    # This assumes the all the features we don't want to
+    # normalise are all always at the front.
+    new_mean = jnp.zeros_like(mean)
+    new_var = jnp.zeros_like(var)
```

Comment:
- The variance is initially set to 0 to make sure we get the computation right at the beginning. The algorithm needs to track the variance. For the first batch we don't have any previous batch so the current variance is 0. But we can not use a variance of 0 for normalisation so the initial standard deviation is set to 1. Then after every computation the standard deviation will be modified to be the sqrt of the variance. 

- The initial count is set to a value close to 0 mainly to avoid any potential issue of having a division by zero.

Reply:
Sounds good :+1: